### PR TITLE
Updated Dutch translations

### PR DESCRIPTION
**Pull request by Bwko**
As requested by  [@tboerger](https://github.com/go-gitea/gitea/pull/487#discussion_r94121404)
cc @mjwwit

### CODE REVIEW

**Comment by andreynering** [comment]
Trusted LGTM

File: options/locale/locale_nl-NL.ini [hunk 1]
```diff
@@ -25,7 +25,7 @@ captcha=CAPTCHA
 
 repository=Repository
 organization=Organisatie
-mirror=Spiegel
+kopie=Kopie
```
Comment (bkcsoft) [inline options/locale/locale_nl-NL.ini]: should be `mirror=Kopie`

**Review by bkcsoft — CHANGES_REQUESTED** [review]
Otherwise Trusted LGTM

File: options/locale/locale_nl-NL.ini [hunk 2]
```diff
@@ -265,7 +265,7 @@ location=Locatie
 update_profile=Profile bijwerken
```
Comment (mjwwit) [inline options/locale/locale_nl-NL.ini]: update_profile=Profiel bijwerken

File: options/locale/locale_nl-NL.ini [hunk 3]
```diff
@@ -25,7 +25,7 @@ captcha=CAPTCHA
 
 repository=Repository
 organization=Organisatie
-mirror=Spiegel
+kopie=Kopie
 new_repo=Nieuwe repository
 new_migrate=Nieuwe migratie
 new_mirror=Nieuwe Kopie
```
Comment (mjwwit) [inline options/locale/locale_nl-NL.ini]: new_mirror=Nieuwe kopie

File: options/locale/locale_nl-NL.ini [hunk 4]
```diff
@@ -59,18 +59,18 @@ db_helper=Gebruik InnoDB engine met utf8_general_ci karakterset voor MySQL.
 ssl_mode=SSL-modus
 path=Pad
 sqlite_helper=Het pad van de SQLite3- of TiDB-database.<br>Als u Gitea start als een service, geef dan een absoluut pad op.
-err_empty_db_path=SQLite3 of TiDB databankpad mag niet leeg.
-err_invalid_tidb_name=TiDB databank naam niet tekens kunnen "." en "-".
+err_empty_db_path=SQLite3 of TiDB database mag niet leeg.
```
Comment (mjwwit) [inline options/locale/locale_nl-NL.ini]: err_empty_db_path=SQLite3 of TiDB database pad mag niet leeg zijn.

File: options/locale/locale_nl-NL.ini [hunk 5]
```diff
@@ -59,18 +59,18 @@ db_helper=Gebruik InnoDB engine met utf8_general_ci karakterset voor MySQL.
...
 no_admin_and_disable_registration=Je kunt niet de registratie uit te schakelen zonder een beheerders account.
-err_empty_admin_password=Beheerder wachtwoord kan niet leeg zijn.
+err_empty_admin_password=Beheerder wachtwoord mag niet leeg zijn.
 
 general_title=Toepassing algemene instellingen
 app_name=Applicatienaam
-app_name_helper=Plaats hier je organisatienaam in grote letters!
-repo_path=Repositories basis directorie
-repo_path_helper=Alle remote Git repositories worden in deze directorie opgeslagen
+app_name_helper=Plaats hier je organisatienaam in hoofdletters!
```
Comment (mjwwit) [inline options/locale/locale_nl-NL.ini]: app_name_helper=Plaats hier uw organisatienaam in hoofdletters!

File: options/locale/locale_nl-NL.ini [hunk 6]
```diff
@@ -297,13 +297,13 @@ email_deletion_desc=Dit e-mailadres verwijdert, worden gerelateerde informatie v
...
 add_email=E-mailadres toevoegen
-add_email_confirmation_sent=Een nieuwe bevestiging e-mail werd verstuurd naar '%s', gelieve uw inbox in de komende %d uren te controleren om het bevestigingsproces te voltooien.
+add_email_confirmation_sent=Een nieuwe bevestigings e-mail werd verstuurd naar '%s', gelieve uw inbox in de komende %d uren te controleren om het bevestigingsproces te voltooien.
 add_email_success=Het e-mailadres was toegevoegd.
 
 manage_ssh_keys=Beheer SSH sleutels
 add_key=Sleutel toevoegen
 ssh_desc=Dit is een lijst van alle SSH sleutels die gekoppeld zijn aan uw account. Verwijder alle sleutels die u niet herkent.
-ssh_helper=<strong>De behoeftehulp van?</strong> Check out onze gids voor het <a href="%s"> genereren van SSH sleutels</a> of <a href="%s"> algemene SSH</a> problemen.
+ssh_helper=<strong>Weet je niet hoe?</strong> Lees dan onze handleiding voor het <a href="%s"> genereren van SSH sleutels</a> of voor <a href="%s"> algemene SSH</a> problemen.
```
Comment (mjwwit) [inline options/locale/locale_nl-NL.ini]: ssh_helper=<strong>Weet u niet hoe?</strong> Lees dan onze handleiding voor het <a href="%s"> genereren van SSH sleutels</a> of voor <a href="%s"> algemene SSH</a> problemen.

**Review by mjwwit — CHANGES_REQUESTED** [review]
Loads better than before! I made a few small suggestions to make it more consistent.

**Comment by Bwko** [comment]
@mjwwit & @bkcsoft Updated

**Review by bkcsoft — APPROVED** [review]

**Comment by bkcsoft** [comment]
I'll have @mjwwit LG_TM since he apparently knows the language at hand 😉

**Comment by mjwwit** [comment]
LGTM

**Comment by bkcsoft** [comment]
something is wrong with LGTM...

**Pull request merged**